manage.py

```diff
@@ -88,8 +88,10 @@ def clear_memcached():
               help="Skip database creation step.")
 @click.option("--test-db", "-t", is_flag=True,
               help="Initialize the test database.")
+@click.option("--force", "-f", is_flag=True,
+              help="Drop existing database and user.")
```

Comment:
This doesn't drop the existing database and the user. Using this option, according to this implementation, drops all tables and types.